client/js/components/TabView/TabView.js

```diff
@@ -0,0 +1,117 @@
+import React, {Component} from 'react';
+import {View, Text, StyleSheet, Image} from 'react-native';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import Browse from '../../screens/Browse';
+import Faves from '../Faves';
+import styles from './styles';
+
+class TabViewProfile extends Component {
+  constructor() {
+    super();
+    this.state = {
+      items: true,
+      store: false,
+      brands: false,
+      myphotos: false,
+    };
+  }
+
+  onChangeStyle(tab) {
+    this.setState({
+      items: false,
+      store: false,
+      brands: false,
+      myphotos: false,
+    });
+    this.setState({[tab]: true});
```

Comment:
This should not work. How are you resetting a tab from true to false?

Reply:
I think shes actually resetting the whole state on every change... and then uopdating the tab to be true